django_select2/util.py

```diff
@@ -101,7 +106,10 @@ def register_field(key, field):
         if GENERATE_RANDOM_ID:
             id_ = u"%d:%s" % (len(__id_store), unicode(datetime.datetime.now()))
         else:
-            id_ = unicode(hashlib.sha1("%s:%s" % (key, SECRET_SALT)).hexdigest())
+            if six.PY3:
```

Comment:
Python 3, was throwing an error that strings needed to be encoded before hashing them. I figured it was simpler to leave Python 2 implementation as is, and just fix the Python 3 way. Would you prefer and single uniform implementation?